### PR TITLE
Try making PyPy 7.3.9

### PR DESCRIPTION
**Pull request by lithomas1**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lithomas1** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,7 +42,7 @@ source:
     folder: pypy2-binary                                                      # [win]
 
 build:
-  number: 2
+  number: 0
```
Comment (mattip) [inline recipe/meta.yaml:45]: I think this should be 0 since it is the first 7.3.9 build

**Comment by mattip** [comment]
The remaining patches will require some adjustment. One way to get this going would be to locally run 
```
conda build -m .ci_support/linux_64_name_suffix3.8openssl1.1.1.yaml recipe
```

and 
```
conda build -m .ci_support/linux_64_name_suffix3.9openssl1.1.1.yaml recipe
```

and then hack away until the patching succeeds.

**Comment by lithomas1** [comment]
@conda-forge-admin, please rerender

**Comment by mattip** [comment]
Thanks

**Comment by lithomas1** [comment]
Hmm. It doesn't look like the packages got uploaded for this PR, and I'm not sure what went wrong. Maybe re-triggering the CI will do it?

**Comment by h-vetinari** [comment]
Unfortunately this has been happening occasionally. A manual restart in Azure can only be done by core members, but it works to push a dummy commit (e.g. `git commit --allow-empty -m "retrigger CI"`) to main (can be done by feedstock maintainers).

Also - depending on the importance of 7.3.9 vs. 7.3.8 -, we should kindly ask who can rebuild aarch/ppc, because those builds cannot be done in CI (previously this was always done by @isuruf AFAIK).

**Comment by mattip** [comment]
I would suggest holding off with the aarch64/ppc builds and rerunning this. Due to [an issue with GenericAlias](https://foss.heptapod.net/pypy/pypy/-/issues/3720), which PyPy fixed yesterday, I will add a new patch here.

**Pull request merged**